google-youtube-upload.html

```diff
@@ -393,6 +422,30 @@
         value: Boolean('MediaRecorder' in window &&
                        'mediaDevices' in navigator &&
                        'getUserMedia' in navigator.mediaDevices)
+      },
+
+      /**
+       * The [MediaStream audio constraints](https://developer.mozilla.org/docs/Web/API/MediaStreamConstraints/audio).
+       */
+      audio: {
+        type: Object,
+        value: true
+      },
+
+      /**
+       * The [MediaStream video constraints](https://developer.mozilla.org/docs/Web/API/MediaStreamConstraints/video).
+       */
+      video: {
+        type: Object,
+        value: true
+      },
+
+      /**
+       * The media type.
+       */
+      mimeType: {
+        type: String,
+        value: ''
```

Comment:
The value should be what I recommended before; If the browser doesn't support those codecs, then the user of this element can change this value. It was `'video/webm;codecs=vp9,vp8'` before, so expectation was not broken.